configs/rhel-sst-csi-client-tools--all.yaml

```diff
@@ -22,6 +22,7 @@ data:
           dependencies:
             - coreutils
             - gpg
+            - insights-core-selinux
```

Comment:
Drop the dependency here, CR doesn't handle placeholders as dependencies.

Reply:
Hi @yselkowitz - What we want to do is to ensure the new `insights-core-selinux` one of the dependencies of the `insights-client`. (i.e., when installing `insights-client` package, the `insights-core-selinux` should be installed at the same time). Will removing this line from the `dependencies` break this dependency relationship? 

Sorry for the first grade question.

Reply:
You will still need the dependency in the spec file, that is what actually matters.  CR can't currently represent that though, so it needs to be omitted here.

Reply:
Got it. Just removed it. Thanks for the clarification.